common/src/commonMain/kotlin/com/epam/drill/common/agent/transport/AgentMessageDestination.kt

```diff
@@ -1,25 +1,28 @@
-/**
- * Copyright 2020 - 2022 EPAM Systems
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.epam.drill.common.ws.dto
-
-import kotlinx.serialization.Serializable
-
-@Serializable
-data class PluginAction(
-    val id: String,
-    val message: String,
-    val confirmationKey: String = "",
-)
+/**
+ * Copyright 2020 - 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.epam.drill.common.agent.transport
+
+/**
+ * Simple data class to hold information about message destination.
+ *
+ * It's used with [AgentMessageSender].
+ *
+ * @see [AgentMessageSender]
+ */
+data class AgentMessageDestination(
+    val type: String,

```

Comment:
For example, topic/queue in case of JMS or other messaging. For theoretically calls when target is entity and type is operation (our REST is example).